### PR TITLE
Issue302 baseline step size

### PR DESCRIPTION
**Pull request by JavierArroyoBastida**
This is for https://github.com/ibpsa/project1-boptest/issues/302 and would also close https://github.com/ibpsa/project1-boptest/issues/240.

### CODE REVIEW

**Comment by JavierArroyoBastida** [comment]
Test case references require an update, which was expected... I'll wait for https://github.com/ibpsa/project1-boptest/pull/192.

File: testcase.py [hunk 1]
```diff
@@ -97,6 +99,8 @@ def __initilize_data(self):
         for key in self.input_names:
             self.u[key] = np.array([])
         self.u_store = copy.deepcopy(self.u)
+        # Results filtering for pyfmi
+        self.options['filter'] = self.output_names + self.input_names
```
Comment (dhblum) [inline testcase.py:103]: Should the filter option be set in the __init__ with the other options?  This just may be more consistent to have all the options set in the __init__ when they can be (basically move this to after line 60, and move call of __initialize_data accordingly in __init__).

**Review by dhblum — CHANGES_REQUESTED** [review]
I think this looks good in general, just the one comment.

**Comment by dhblum** [comment]
Just a note too, the latest travis check on this PR took about 30 minutes less time than the travis check from the last merged PR, and the reference results directory in this PR is half the size of the one currently on maser.  Good stuff!

File: testcase.py (hunk 1 above)
Comment (JavierArroyoBastida) [reply testcase.py]: I think that totally makes sense. Implemented in https://github.com/ibpsa/project1-boptest/pull/324/commits/b9110650339c09bf2e80c00cbb7d04b711760a41.

**Comment by JavierArroyoBastida** [comment]
That's great news indeed!

**Review by dhblum — APPROVED** [review]
Looks good!  Will merge.  Thanks again.

**Pull request merged**